File: Julia/FizzBuzz/fizzbuzz.jl
function fizzbuzz(i)
    if i % 15 == 0 "FizzBuzz"
    elseif i % 3 == 0 "Fizz"
    elseif i % 5 == 0 "Buzz"
    else i
    end
end

function run(n)
    for i = 1:n
        println(fizzbuzz(i))
    end
end

run(15)

File: R/FizzBuzz/fizzbuzz.r
run <- function(n) {
  for(i in 1:n) {
    if (i %% 15 == 0) {
      print("FizzBuzz")
    } else if (i %% 3 == 0) {
      print("Fizz")
    } else if (i %% 5 == 0) {
      print("Buzz")
    } else {
      print(i)
    }
  }
}

run(15)
